src/main/java/de/tum/in/www1/artemis/service/connectors/gitlab/dto/GitLabPersonalAccessTokenRequestDTO.java

```diff
@@ -8,59 +8,7 @@
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public class GitLabPersonalAccessTokenRequestDTO {
+public record GitLabPersonalAccessTokenRequestDTO(@JsonProperty String name, @JsonProperty("user_id") Long userId, @JsonProperty("scopes") String[] scopes,
+        @JsonProperty("expires_at") Date expiresAt) {
```

Comment:
I think we can just remove the `@JsonProperty` annotations since they don't do anything, right?
```suggestion
public record GitLabPersonalAccessTokenRequestDTO(String name, @JsonProperty("user_id") Long userId, String[] scopes,
        @JsonProperty("expires_at") Date expiresAt) {
```